scripts/asr/transcribe_file_offline.py

```diff
@@ -31,7 +31,7 @@ def main() -> None:
     config = riva.client.RecognitionConfig(
         encoding=riva.client.AudioEncoding.LINEAR_PCM,
         language_code=args.language_code,
-        max_alternatives=1,
+        max_alternatives=args.max_alternatives,
```

Comment:
It doesn't look like `print_offline`  uses `max_alternatives`. Let's revert this change for now.  We can add it later.

Reply:
looks like general fix to address the hardcoded 1?